functional/FunctionalTest.java

```diff
@@ -421,6 +422,13 @@ private static void handleException(String methodName, String args, long startTi
     throw e;
   }
 
+  private static void deleteFilesAndObjects(String bucketName, String files[]) throws Exception {
+    for (String filename : files) {
+      Files.delete(Paths.get(filename));
+      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename).build());
```

Comment:
This method is not necessary. you could use `removeObjects()` then this `for`

Reply:
This method is used to make the functional Test less verbose as the same thing is happening in 6 `composeObject` test and it also deletes the files. I have used `removeObjects` now. This